README.md

```diff
@@ -379,3 +379,5 @@ the created run configuration to your needs, e.g. by adding an argument and opti
 Copyright (C) 2017-2020 HERE Europe B.V.
 
 See the [LICENSE](./LICENSE) file in the root of this project for license details.
+
+OSS Review Toolkit (ORT) is a [Linux Foundation project](https://www.linuxfoundation.org).
```

Comment:
Commit message sounds weird, what about "README: Mention that ORT is a LF project"?

Reply:
Changed it "README: Add line stating ORT is a LF project" but I am open to suggestions for a better commit message.